lib/travis/build/addons/deploy/script.rb

```diff
@@ -203,7 +204,7 @@ def options
             end
 
             def warning_message(message)
-              sh.echo "Skipping a deployment with the #{config[:provider]} provider because #{message}", ansi: :yellow
+              sh.echo "Skipping a deployment with the #{provider} provider because #{message}", ansi: :yellow
```

Comment:
Ruby n00b here but should we refer to `provider` as `@provider`? 🤔

Reply:
Using the accessor makes it easier to change presentation later if need be, with `def provider`.

Reply:
Got it! Thanks!